tensorflow/contrib/cmake/tf_tests.cmake

```diff
@@ -148,14 +148,28 @@ if (tensorflow_BUILD_PYTHON_TESTS)
     "${tensorflow_source_dir}/tensorflow/python/training/*_test.py"
     "${tensorflow_source_dir}/tensorflow/contrib/data/*_test.py"
     "${tensorflow_source_dir}/tensorflow/contrib/factorization/*_test.py"
-    "${tensorflow_source_dir}/tensorflow/contrib/keras/python/keras/integration_test.py"
```

Comment:
Was this test failing? It would be better to leave it here if not, so that it continues to be part of our presubmits...